2025-06-19 23:11:10,574 INFO    Thread-8 (_run_job):150799 [wandb_setup.py:_flush():70] Current SDK version is 0.19.11
2025-06-19 23:11:10,574 INFO    Thread-8 (_run_job):150799 [wandb_setup.py:_flush():70] Configure stats pid to 150799
2025-06-19 23:11:10,574 INFO    Thread-8 (_run_job):150799 [wandb_setup.py:_flush():70] Loading settings from /home/cagri/.config/wandb/settings
2025-06-19 23:11:10,574 INFO    Thread-8 (_run_job):150799 [wandb_setup.py:_flush():70] Loading settings from /home/cagri/Desktop/IS-584-Term-Project/wandb/settings
2025-06-19 23:11:10,574 INFO    Thread-8 (_run_job):150799 [wandb_setup.py:_flush():70] Loading settings from environment variables
2025-06-19 23:11:10,574 INFO    Thread-8 (_run_job):150799 [wandb_init.py:setup_run_log_directory():724] Logging user logs to /home/cagri/Desktop/IS-584-Term-Project/wandb/run-20250619_231110-irrx7qgd/logs/debug.log
2025-06-19 23:11:10,574 INFO    Thread-8 (_run_job):150799 [wandb_init.py:setup_run_log_directory():725] Logging internal logs to /home/cagri/Desktop/IS-584-Term-Project/wandb/run-20250619_231110-irrx7qgd/logs/debug-internal.log
2025-06-19 23:11:10,574 INFO    Thread-8 (_run_job):150799 [wandb_init.py:monkeypatch_ipython():643] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f0c46cc3500>
2025-06-19 23:11:10,575 INFO    Thread-8 (_run_job):150799 [wandb_init.py:init():852] calling init triggers
2025-06-19 23:11:10,575 INFO    Thread-8 (_run_job):150799 [wandb_init.py:init():857] wandb.init called with sweep_config: {'learning_rate': 2e-05, 'lora_dropout': 0.05, 'r': 1}
config: {'_wandb': {}}
2025-06-19 23:11:10,575 INFO    Thread-8 (_run_job):150799 [wandb_init.py:init():893] starting backend
2025-06-19 23:11:10,575 INFO    Thread-8 (_run_job):150799 [wandb_init.py:init():897] sending inform_init request
2025-06-19 23:11:10,576 INFO    Thread-8 (_run_job):150799 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-06-19 23:11:10,576 INFO    Thread-8 (_run_job):150799 [wandb_init.py:init():907] backend started and connected
2025-06-19 23:11:10,577 INFO    Thread-8 (_run_job):150799 [wandb_run.py:_config_callback():1436] config_cb None None {'learning_rate': 2e-05, 'lora_dropout': 0.05, 'r': 1}
2025-06-19 23:11:10,578 INFO    Thread-8 (_run_job):150799 [wandb_run.py:_label_probe_notebook():1376] probe notebook
2025-06-19 23:11:10,579 INFO    Thread-8 (_run_job):150799 [wandb_init.py:init():1005] updated telemetry
2025-06-19 23:11:10,582 INFO    Thread-8 (_run_job):150799 [wandb_init.py:init():1029] communicating run to backend with 90.0 second timeout
2025-06-19 23:11:11,143 INFO    Thread-8 (_run_job):150799 [wandb_init.py:init():1104] starting run threads in backend
2025-06-19 23:11:11,238 INFO    Thread-8 (_run_job):150799 [wandb_run.py:_console_start():2573] atexit reg
2025-06-19 23:11:11,239 INFO    Thread-8 (_run_job):150799 [wandb_run.py:_redirect():2421] redirect: wrap_raw
2025-06-19 23:11:11,239 INFO    Thread-8 (_run_job):150799 [wandb_run.py:_redirect():2490] Wrapping output streams.
2025-06-19 23:11:11,239 INFO    Thread-8 (_run_job):150799 [wandb_run.py:_redirect():2513] Redirects installed.
2025-06-19 23:11:11,240 INFO    Thread-8 (_run_job):150799 [wandb_init.py:init():1150] run started, returning control to user process
2025-06-19 23:11:16,270 INFO    Thread-8 (_run_job):150799 [wandb_run.py:_config_callback():1436] config_cb None None {'output_dir': './results', 'overwrite_output_dir': False, 'do_train': False, 'do_eval': False, 'do_predict': False, 'eval_strategy': 'no', 'prediction_loss_only': False, 'per_device_train_batch_size': 1, 'per_device_eval_batch_size': 1, 'per_gpu_train_batch_size': None, 'per_gpu_eval_batch_size': None, 'gradient_accumulation_steps': 1, 'eval_accumulation_steps': None, 'eval_delay': 0, 'torch_empty_cache_steps': None, 'weight_decay': 0.0, 'adam_beta1': 0.9, 'adam_beta2': 0.999, 'adam_epsilon': 1e-08, 'max_grad_norm': 1.0, 'num_train_epochs': 5, 'max_steps': -1, 'lr_scheduler_type': 'linear', 'lr_scheduler_kwargs': {}, 'warmup_ratio': 0.0, 'warmup_steps': 0, 'log_level': 'passive', 'log_level_replica': 'warning', 'log_on_each_node': True, 'logging_dir': './logs', 'logging_strategy': 'steps', 'logging_first_step': False, 'logging_steps': 100, 'logging_nan_inf_filter': True, 'save_strategy': 'steps', 'save_steps': 500, 'save_total_limit': 1, 'save_safetensors': True, 'save_on_each_node': False, 'save_only_model': False, 'restore_callback_states_from_checkpoint': False, 'no_cuda': False, 'use_cpu': False, 'use_mps_device': False, 'seed': 42, 'data_seed': None, 'jit_mode_eval': False, 'use_ipex': False, 'bf16': False, 'fp16': True, 'fp16_opt_level': 'O1', 'half_precision_backend': 'auto', 'bf16_full_eval': False, 'fp16_full_eval': False, 'tf32': None, 'local_rank': 0, 'ddp_backend': None, 'tpu_num_cores': None, 'tpu_metrics_debug': False, 'debug': [], 'dataloader_drop_last': False, 'eval_steps': None, 'dataloader_num_workers': 0, 'dataloader_prefetch_factor': None, 'past_index': -1, 'run_name': 'term-project', 'disable_tqdm': False, 'remove_unused_columns': True, 'label_names': None, 'load_best_model_at_end': False, 'metric_for_best_model': None, 'greater_is_better': None, 'ignore_data_skip': False, 'fsdp': [], 'fsdp_min_num_params': 0, 'fsdp_config': {'min_num_params': 0, 'xla': False, 'xla_fsdp_v2': False, 'xla_fsdp_grad_ckpt': False}, 'fsdp_transformer_layer_cls_to_wrap': None, 'accelerator_config': {'split_batches': False, 'dispatch_batches': None, 'even_batches': True, 'use_seedable_sampler': True, 'non_blocking': False, 'gradient_accumulation_kwargs': None}, 'deepspeed': None, 'label_smoothing_factor': 0.0, 'optim': 'adamw_torch', 'optim_args': None, 'adafactor': False, 'group_by_length': False, 'length_column_name': 'length', 'report_to': ['wandb'], 'ddp_find_unused_parameters': None, 'ddp_bucket_cap_mb': None, 'ddp_broadcast_buffers': None, 'dataloader_pin_memory': True, 'dataloader_persistent_workers': False, 'skip_memory_metrics': True, 'use_legacy_prediction_loop': False, 'push_to_hub': False, 'resume_from_checkpoint': None, 'hub_model_id': None, 'hub_strategy': 'every_save', 'hub_token': '<HUB_TOKEN>', 'hub_private_repo': None, 'hub_always_push': False, 'gradient_checkpointing': False, 'gradient_checkpointing_kwargs': None, 'include_inputs_for_metrics': False, 'include_for_metrics': [], 'eval_do_concat_batches': True, 'fp16_backend': 'auto', 'push_to_hub_model_id': None, 'push_to_hub_organization': None, 'push_to_hub_token': '<PUSH_TO_HUB_TOKEN>', 'mp_parameters': '', 'auto_find_batch_size': False, 'full_determinism': False, 'torchdynamo': None, 'ray_scope': 'last', 'ddp_timeout': 1800, 'torch_compile': False, 'torch_compile_backend': None, 'torch_compile_mode': None, 'include_tokens_per_second': False, 'include_num_input_tokens_seen': False, 'neftune_noise_alpha': None, 'optim_target_modules': None, 'batch_eval_metrics': False, 'eval_on_start': False, 'use_liger_kernel': False, 'eval_use_gather_object': False, 'average_tokens_across_devices': False}
2025-06-20 05:30:58,403 INFO    MainThread:150799 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-06-20 05:31:09,452 INFO    Thread-8 (_run_job):150799 [wandb_run.py:_finish():2321] finishing run e274028-metu-middle-east-technical-university/tinyllama-lora/irrx7qgd
2025-06-20 05:31:09,453 INFO    Thread-8 (_run_job):150799 [jupyter.py:save_history():477] not saving jupyter history
2025-06-20 05:31:09,453 INFO    Thread-8 (_run_job):150799 [jupyter.py:save_ipynb():401] not saving jupyter notebook
2025-06-20 05:31:09,453 INFO    Thread-8 (_run_job):150799 [wandb_init.py:_jupyter_teardown():628] cleaning up jupyter logic
2025-06-20 05:31:09,453 INFO    Thread-8 (_run_job):150799 [wandb_run.py:_atexit_cleanup():2538] got exitcode: 1
2025-06-20 05:31:09,453 INFO    Thread-8 (_run_job):150799 [wandb_run.py:_restore():2520] restore
2025-06-20 05:31:09,453 INFO    Thread-8 (_run_job):150799 [wandb_run.py:_restore():2526] restore done
2025-06-20 05:31:09,453 INFO    Thread-8 (_run_job):150799 [wandb_run.py:_restore():2520] restore
2025-06-20 05:31:09,453 INFO    Thread-8 (_run_job):150799 [wandb_run.py:_restore():2526] restore done
2025-06-20 05:31:09,453 ERROR   Thread-8 (_run_job):150799 [wandb_run.py:_atexit_cleanup():2559] Problem finishing run
Traceback (most recent call last):
  File "/home/cagri/.local/lib/python3.12/site-packages/wandb/agents/pyagent.py", line 306, in _run_job
    self._function()
  File "/tmp/ipykernel_150799/1012406194.py", line 88, in sweep_train
    trainer.train()
  File "/home/cagri/.local/lib/python3.12/site-packages/transformers/trainer.py", line 2240, in train
    return inner_training_loop(
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/cagri/.local/lib/python3.12/site-packages/transformers/trainer.py", line 2621, in _inner_training_loop
    self.control = self.callback_handler.on_step_end(args, self.state, self.control)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/cagri/.local/lib/python3.12/site-packages/transformers/trainer_callback.py", line 534, in on_step_end
    return self.call_event("on_step_end", args, state, control)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/cagri/.local/lib/python3.12/site-packages/transformers/trainer_callback.py", line 556, in call_event
    result = getattr(callback, event)(
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/cagri/.local/lib/python3.12/site-packages/transformers/utils/notebook.py", line 310, in on_step_end
    self.training_tracker.update(
  File "/home/cagri/.local/lib/python3.12/site-packages/transformers/utils/notebook.py", line 166, in update
    self.update_bar(value)
  File "/home/cagri/.local/lib/python3.12/site-packages/transformers/utils/notebook.py", line 191, in update_bar
    self.display()
  File "/home/cagri/.local/lib/python3.12/site-packages/transformers/utils/notebook.py", line 234, in display
    self.output.update(disp.HTML(self.html_code))
  File "/home/cagri/.local/lib/python3.12/site-packages/IPython/core/display_functions.py", line 354, in update
    update_display(obj, display_id=self.display_id, **kwargs)
  File "/home/cagri/.local/lib/python3.12/site-packages/IPython/core/display_functions.py", line 306, in update_display
    display(obj, display_id=display_id, **kwargs)
  File "/home/cagri/.local/lib/python3.12/site-packages/IPython/core/display_functions.py", line 285, in display
    publish_display_data(data=format_dict, metadata=md_dict, **kwargs)
  File "/home/cagri/.local/lib/python3.12/site-packages/IPython/core/display_functions.py", line 73, in publish_display_data
    display_pub.publish(
  File "/home/cagri/.local/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 654, in publish
    ipython.display_pub._orig_publish(data, metadata=metadata, **kwargs)
  File "/home/cagri/.local/lib/python3.12/site-packages/ipykernel/zmqshell.py", line 103, in publish
    self._flush_streams()
  File "/home/cagri/.local/lib/python3.12/site-packages/ipykernel/zmqshell.py", line 66, in _flush_streams
    sys.stdout.flush()
  File "/home/cagri/.local/lib/python3.12/site-packages/ipykernel/iostream.py", line 604, in flush
    self.pub_thread.schedule(self._flush)
  File "/home/cagri/.local/lib/python3.12/site-packages/ipykernel/iostream.py", line 267, in schedule
    self._event_pipe.send(b"")
  File "/home/cagri/.local/lib/python3.12/site-packages/zmq/sugar/socket.py", line 698, in send
    return super().send(data, flags=flags, copy=copy, track=track)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "_zmq.py", line 1081, in zmq.backend.cython._zmq.Socket.send
  File "_zmq.py", line 1123, in zmq.backend.cython._zmq.Socket.send
  File "_zmq.py", line 1268, in zmq.backend.cython._zmq._check_closed
zmq.error.ZMQError: Socket operation on non-socket

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/cagri/.local/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 2550, in _atexit_cleanup
    self._on_finish()
  File "/home/cagri/.local/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 2801, in _on_finish
    with progress.progress_printer(
  File "/usr/lib/python3.12/contextlib.py", line 137, in __enter__
    return next(self.gen)
           ^^^^^^^^^^^^^^
  File "/home/cagri/.local/lib/python3.12/site-packages/wandb/sdk/lib/progress.py", line 92, in progress_printer
    with printer.dynamic_text() as text_area:
  File "/usr/lib/python3.12/contextlib.py", line 137, in __enter__
    return next(self.gen)
           ^^^^^^^^^^^^^^
  File "/home/cagri/.local/lib/python3.12/site-packages/wandb/sdk/lib/printer.py", line 465, in dynamic_text
    handle = self._ipython_display.display(
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/cagri/.local/lib/python3.12/site-packages/IPython/core/display_functions.py", line 285, in display
    publish_display_data(data=format_dict, metadata=md_dict, **kwargs)
  File "/home/cagri/.local/lib/python3.12/site-packages/IPython/core/display_functions.py", line 73, in publish_display_data
    display_pub.publish(
  File "/home/cagri/.local/lib/python3.12/site-packages/ipykernel/zmqshell.py", line 103, in publish
    self._flush_streams()
  File "/home/cagri/.local/lib/python3.12/site-packages/ipykernel/zmqshell.py", line 66, in _flush_streams
    sys.stdout.flush()
  File "/home/cagri/.local/lib/python3.12/site-packages/ipykernel/iostream.py", line 604, in flush
    self.pub_thread.schedule(self._flush)
  File "/home/cagri/.local/lib/python3.12/site-packages/ipykernel/iostream.py", line 267, in schedule
    self._event_pipe.send(b"")
  File "/home/cagri/.local/lib/python3.12/site-packages/zmq/sugar/socket.py", line 698, in send
    return super().send(data, flags=flags, copy=copy, track=track)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "_zmq.py", line 1081, in zmq.backend.cython._zmq.Socket.send
  File "_zmq.py", line 1123, in zmq.backend.cython._zmq.Socket.send
  File "_zmq.py", line 1268, in zmq.backend.cython._zmq._check_closed
zmq.error.ZMQError: Socket operation on non-socket
2025-06-20 05:33:57,143 INFO    MsgRouterThr:150799 [mailbox.py:close():129] [no run ID] Closing mailbox, abandoning 1 handles.
